minimap2/2.26/Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM ubuntu:jammy as app
+
+# for easy upgrade later. ARG variables only persist during image build time
+ARG MINIMAP2_VER="2.26"
+
+# metadata
+LABEL base.image="ubuntu:jammy"
+LABEL dockerfile.version="1"
+LABEL software="Minimap2"
+LABEL software.version="${MINIMAP2_VER}"
+LABEL description="versatile sequence alignment program that aligns DNA or mRNA sequences against a large reference database"
+LABEL website="https://github.com/lh3/minimap2"
+LABEL license="https://github.com/lh3/minimap2/blob/master/LICENSE.txt"
+LABEL maintainer="Kelsey Florek"
+LABEL maintainer.email="Kelsey.florek@slh.wisc.edu"
+LABEL maintainer2="Curtis Kapsak"
+LABEL maintainer2.email="kapsakcj@gmail.com"
+
+# install deps and cleanup apt garbage
+RUN apt-get update && apt-get install -y --no-install-recommends \
+ curl \
+ ca-certificates \
+ bzip2 \
+ procps && \
+ apt-get autoclean && rm -rf /var/lib/apt/lists/*
+
+# install minimap2 binary; make /data
+RUN curl -L https://github.com/lh3/minimap2/releases/download/v${MINIMAP2_VER}/minimap2-${MINIMAP2_VER}_x64-linux.tar.bz2 | tar -jxvf - && \
+ mkdir /data
+
+# set final PATH and LC_ALL for singularity compatibility
+ENV PATH="${PATH}:/minimap2-${MINIMAP2_VER}_x64-linux" \
+LC_ALL=C
+
```

Comment:
This looks great!

Can you add a CMD instruction to the app stage? This the default command that is run in the container. For STaPH-B, we like to use the tool's help option.

For minimap2 it'd be something like

```
CMD minimap2 -h
```

Reply:
Done!